### ISSUE TITLE
🐛 Bug: Infinite loop in Cypress

### ISSUE THREAD

**Issue by TheWathis** [tracim/tracim#6097]
### Tracim version

4.4.3_build_099 (with the mention development (not merged yet))

### Issue description

During Cypress tests, sometimes tests are looping on an API call to fetch the user avatar (since the avatar is now generated by the backend)

### How to reproduce

Start one of theses tests:
- click_on_sidebar_titles_spec.js
- sidebar_buttons_spec.js

### Related issue

#2872

**Comment by Skylsmoi** [comment]
we have tested that if upgrading cypress to latest version, issue cannot be reproduced

**Comment by Haskaris** [comment]
So, just to be sure to understand, you have upgraded Cypress to 12+?

**Comment by Skylsmoi** [comment]
nope, I'm just stating that during our investigation, we tried to reproduce the issue with cypress 12 (not sure about the exact version) and we couldn't.

Is that right?

**Comment by Haskaris** [comment]
Yes you're right
See https://github.com/tracim/tracim/issues/2872#issuecomment-1424375036

**Comment by Skylsmoi** [comment]
We are now using Cypress 13+.

Haven't seen the issue again, closing the issue
